drivers/ethernet/eth_enc424j600.c

```diff
@@ -556,6 +556,41 @@ static enum ethernet_hw_caps enc424j600_get_capabilities(const struct device *de
 	return ETHERNET_LINK_10BASE_T | ETHERNET_LINK_100BASE_T;
 }
 
+static int enc424j600_set_config(const struct device *dev,
+                                 enum ethernet_config_type type,
+                                 const struct ethernet_config *config)
```

Comment:
Please stick to the used indentation elsewhere (tabs + aligning spaces)

```suggestion
static int enc424j600_set_config(const struct device *dev,
				 enum ethernet_config_type type,
				 const struct ethernet_config *config)
```

Reply:
Well then i got an error that the line is to long.. I just copied everything from the get_config function so why do I need to fix the indent in the first place?

Reply:
You had too many tabs (= 8 spaces) in the previous version.

Reply:
And apparently I cannot use the button commit suggestions.........................